frontend/src/utils/validation/email.js

```diff
@@ -0,0 +1,5 @@
+import { common } from "./common"
+
+const REGEX = /^[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*.[a-zA-Z]{2,3}$/i
```

Comment:
https://regexr.com/ 에서 테스트해보니
`a@acom`처럼 "."을 빼먹어도 제대로 동작하네요

```suggestion
const REGEX = /^[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*@[0-9a-zA-Z]([-_.]?[0-9a-zA-Z])*\.[a-zA-Z]{2,3}$/i
```

Reply:
`abc_@abc.com` `_`가 제일 뒤에 있으면 안되는 것 같아요.

[전자 우편 주소 - 위키](https://ko.wikipedia.org/wiki/%EC%A0%84%EC%9E%90_%EC%9A%B0%ED%8E%B8_%EC%A3%BC%EC%86%8C)

문법을 완전히 따를 필요는 없지만 주요 메일 서버의 이메일 형식을 한번 확인해보는 게 좋을 것 같습니다.